### DISCUSSION TITLE
Facing Issues with JIT mode.

### DISCUSSION THREAD

**Discussion by PravunathSingh** [tailwindlabs/tailwindcss#4451] [Help]
Recently I have been trying the JIT in my projects but I have been facing some issues.

On enabling the JIT mode and running the 
`"scripts": {
    "dev": "tailwindcss build ./css/tailwind.css -o ./dist/tailwind.css -w"
}` command, even though I have kept the watch mode active the classes like `"font, color and size"` are not being applied.

To apply the classes I manually have to run the `npm run dev` every time I make a change.

It would be really great if someone could help me out with this.

**Comment by tupton** [comment]
[tailwind will automatically watch if `NODE_ENV=development`](https://tailwindcss.com/docs/just-in-time-mode#watch-mode-and-one-off-builds):

> By default, Tailwind will start a long-running watch process if NODE_ENV=development, and it will run in one-off mode if NODE_ENV=production.

Additionally, I believe you have to use the `postcss` CLI to enable watch mode. See #4097.

Try

```json
"scripts": {
  "dev": "NODE_ENV=development postcss tailwindcss -o ./dist/tailwind.css -w"
}
```

**Comment by PravunathSingh** [comment]
Alright, thanks a lot @tupton!! Will try this.
